SwiftReflector/FunctionDeclarationWrapperFinder.cs

```diff
@@ -21,9 +21,28 @@ public FunctionDeclarationWrapperFinder (TypeMapper mapper, WrappingResult wrapp
 		public FunctionDeclaration FindWrapper (FunctionDeclaration original)
 		{
 			var name = original.IsProperty ? original.PropertyName : original.Name;
+
+			var parentOrModuleFull = original.Parent != null ?
+				original.Parent.ToFullyQualifiedName (true) : original.Module.Name;
+			var parentOrModuleBrief = original.Parent != null ?
+				original.Parent.ToFullyQualifiedName (false) : original.Module.Name;
```

Comment:
Minor, but you can use the ?? operator and ?. here:
```suggestion
			var parentOrModuleFull = original.Parent?.ToFullyQualifiedName (true) ?? original.Module.Name;
			var parentOrModuleBrief = original.Parent?.ToFullyQualifiedName (false) ?? original.Module.Name;
```